ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1899,6 +1899,8 @@ ip_igmp_snooping:
 ```yaml
 router_multicast:
   ipv4:
+    counters:
+      rate_period_decay: < 0-600 >
```

Comment:
I'd mark it as optional in README.

Reply:
Please also specify that the interval is in seconds.

Reply:
everything (ish) in eos_cli_config_gen is optional.

Reply:
Yea nothing is mandatory.  I’m confused on the feedback given this fact.

I’m happy to specify the interval is in seconds, but I would need to change the key to something different than what eos calls it.  I do not think that is done anywhere in eos_cli_config_gen

Reply:
To be clear, my suggestion was: `rate_period_decay: < 0-600 >  # optional, seconds`
As we agreed that optional is not required, just: `rate_period_decay: < 0-600 >  # seconds`
Simply a comment providing additional info about the variable.

Reply:
@ryanmerolle

Reply:
Done.  Thanks